include/Point.h

```diff
@@ -82,9 +82,17 @@ namespace influxdb
         /// Fields getter
         std::string getFields() const;
 
+        /// Get fields as a std::deque
+        using FieldsDeque = std::deque<std::pair<std::string, FieldValue>>;
```

Comment:
I'd prefer less encoding of implementation details into the name, at some point the container might change (same for `TagsDeque`). Are simple `Fields` / `Tags` an option?

I don't have strong opinion here though. So feel free to disagree :-).

Reply:
Same applies for the method naming, but there are some more thoughts to consider (see comment).

Reply:
I agree, it is a bit clumsy. Influx themselves use the terminology "[Tag set](https://docs.influxdata.com/influxdb/cloud/reference/syntax/line-protocol/#tag-set)" and "[Field set](https://docs.influxdata.com/influxdb/cloud/reference/syntax/line-protocol/#field-set)" in the Line protocol specification.

That terminology could possibly be confused with `std::set` but I think it's better to use the official nomenclature than try to come up with something else. I'll update accordingly.